### DISCUSSION TITLE
Potentially outdated link in Renovate GitHub Application

### DISCUSSION THREAD

**Discussion by TWiStErRob** [renovatebot/renovate#25584] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I mean this (highlighted in blue) one:
![image](https://github.com/renovatebot/renovate/assets/2906988/d5209302-052a-45c2-afd7-dbfc86c905b4)

https://www.mend.io/free-developer-tools/renovate/ - redirects to -> https://www.mend.io/renovate/
The redirect itself feels like it's outdated, but it still works.

However, based on the URL path, I have a feeling the link is meant to be pointing to https://www.mend.io/renovate-free/ which is accessible through the "Access Free Tools" button:
![image](https://github.com/renovatebot/renovate/assets/2906988/8f094bd2-90ca-4df9-93f6-d805521f0c1f)

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment, marked as answer]
I've changed it to mend.io/renovate (neither page is really perfect)
